src/SimpleEventStore/SimpleEventStore.CosmosDb/AzureCosmosDbStorageEngine.cs

```diff
@@ -11,9 +11,11 @@ namespace SimpleEventStore.CosmosDb
 {
     internal class AzureCosmosDbStorageEngine : IStorageEngine
     {
+        private const int DefaultAutoscaleMaxThroughput = 4000;
```

Comment:
I'm not sure we should be setting a default throughput - while the setting is the minimum supported value, I'd rather have people think about the offer levels they want to use so the cost and throughput trade-offs are taken into consideration.

Reply:
This parameter will be used only when user will not specify the custom collection/database throughput and set the 'UseAutoscale' flag to be True. It's minimum number which can be used for Auto scale mode, so we can not leave it empty or use 0.

If we will not do that, there will be raised an error when 'UseAutoscale' flag will be True:

`Microsoft.Azure.Cosmos.CosmosException: 'Response status code does not indicate success: BadRequest (400); Substatus: 0; ActivityId: c9178727-9310-4d01-b0ff-965406a7e9bb; Reason: (Message: {"Errors":["The offer should have valid throughput values between 4000 and 1000000 inclusive in increments of 1000."]}`